src/styles/utils/_reset.scss

```diff
@@ -0,0 +1,6 @@
+body,
+main,
+div {
```

Comment:
Do we really need to reset margin and padding for div and main?

Reply:
http://html5doctor.com/html-5-reset-stylesheet/ 
I suppose that yes) We need to resel all the elements that we will have in project to avoid that browser will put his own margins on